components/movementsensor/gpsrtkserial/gpsrtkserial.go

```diff
@@ -427,11 +427,7 @@ func (g *rtkSerial) connectAndParseSourceTable() error {
 
 	g.logger.Debug("gettting source table")
 
-	srcReader, err := g.ntripClient.Client.GetSourcetable()
-	g.logger.Debugf("getSourceTable sends: %v\n", srcReader)
-	g.logger.Debugf("error from getsourcetable: %v\n", err)
-
-	srcTable, err := g.ntripClient.Client.ParseSourcetable()
+	srcTable, err := g.ntripClient.ParseSourcetable()
```

Comment:
Do you need to refactor this in the i2c driver as well?

Reply:
Yes. thanks for point this out.

Reply:
Actually that would require a lot of changes. I'll do it in a separate PR